README.md

```diff
@@ -160,12 +164,49 @@ pipenv run test
 Along with running the test suite, the developers of GatorGrader use statement
 and branch coverage to inform their testing activities. To see the coverage of
 the tests while also highlighting the lines that are not currently covered by
-the tests, you can type this command in a terminal window.
+the tests, you can run this command in a terminal window.
```

Comment:
Thanks for the suggestion, if the version of Python that is used is mentioned under Test Coverage, the used version of Python would also have to be mentioned under Automated Testing. Perhaps it makes more sense to leave the comment about this command running with whichever version of Python Pipenv is bound to under Testing With Multiple Versions of Python.

Reply:
I have improved the documentation so that it always stresses the version of Python in which the commands are executed. I think that this should resolve the issue.